tsl/test/sql/compression_errors.sql

```diff
@@ -344,6 +344,7 @@ EXPLAIN SELECT DISTINCT 1 FROM test;
 
 --compress chunks
 SELECT COMPRESS_CHUNK(X) FROM SHOW_CHUNKS('test') X;
+ANALYZE test;
```

Comment:
it's not obvious to me why the ANALYZE?

Reply:
`reindex_relation` was updating the pg_class stats but now they are not updated so without an analyze the results would differ.